frontend/src/app/app.component.html

```diff
@@ -1,35 +1,46 @@
 <md-sidenav-container>
     <md-sidenav class="oc-mobile-sidenav" *ngIf="mobile">
         <md-toolbar>
-            <a [routerLink]="'/'" class="oc-mobile-sidenav-title">{{title}}</a>
+            <a [routerLink]="'/'" class="oc-mobile-sidenav-title text-primary-dark" tabindex="-1">{{title}}</a>
         </md-toolbar>
         <oc-divider></oc-divider>
-        <oc-nav></oc-nav>
+        <oc-nav>
+            <!-- TODO only temporary -->
+            {{'i18n.common.lang.de' | translate}}
+            <md-slide-toggle (change)="changeLang()" [(ngModel)]="isEnglish"></md-slide-toggle>
+            {{'i18n.common.lang.en' | translate}}
+        </oc-nav>
     </md-sidenav>
     <md-toolbar color="primary" class="oc-primary-toolbar">
         <md-icon class="toolbar-icon" (click)="openSidenav()" *ngIf="mobile">menu</md-icon>
         <a routerLink="/">{{title}}</a>
         <span class="toolbar-fill-space"></span>
-        {{'i18n.common.lang.de' | translate}}
-        <md-slide-toggle (change)="changeLang()" [(ngModel)]="isEnglish"></md-slide-toggle>
-        {{'i18n.common.lang.en' | translate}}
-        <button md-button *ngIf="!auth.isLoggedIn()" (click)="login()">Login</button>
-        <button md-button *ngIf="auth.isLoggedIn()" (click)="logout()">Logout</button>
+        <button md-button *ngIf="auth.isLoggedIn(); else notLoggedIn" (click)="logout()">Logout</button>
+        <ng-template #notLoggedIn>
+            <button md-button (click)="login()">Login</button>
+        </ng-template>
         <button md-icon-button [md-menu-trigger-for]="themeMenu">
             <md-icon>more_vert</md-icon>
         </button>
         <md-menu #themeMenu="mdMenu" class="oc-theme-menu">
             <span class="oc-theme-menu-title">{{'i18n.common.theme.title' | translate}}</span>
             <button md-menu-item *ngFor="let theme of allThemes" (click)="changeTheme(theme)">
```

Comment:
This needs some left padding when the navigation is expanded. IMO the slider should only be visible when the navigation is expanded too because it looks very weird when the navigation is minimized.